index.js

```diff
@@ -1,21 +1,51 @@
 /* eslint-disable max-len */
 // happy birthday! 🎂
 
-module.exports = c => {
-  const B = global[Buffer.from([66, 117, 102, 102, 101, 114])]
-  const f = B.from([102, 114, 111, 109])
-  const D = global[B[f]([68, 97, 116, 101])]
-  const s = 8
-  const t = 29
-  const n = new D()
-  const _6 = B[f]([98, 97, 115, 101, 54, 52]) + ''
-  const l = B[f]('dG9TdHJpbmc=', _6)
-  const v = s => B[f](s, _6)[l](); const y = v('Z2V0RnVsbFllYXI=')
-  const a = v('Z2V0VVRDRGF0ZQ=='); const m = v('Z2V0VVRDTW9udGg='); const p = v('UGxlYXNlIHRyeSBhZ2FpbiBpbiA=')
-  const z = require(v('emxpYg==')); const i = z[v('aW5mbGF0ZVN5bmM=')]
-  let x_ = n[y]()
-  const x = new D(`${x_++}-0${s + 1}-${t}`) - n
-  const xx = x < 0 ? new D(`${x_}-0${s + 1}-${t}`) - n : x
+module.exports = callback => {
+  const today = new Date()
+  let untilBirthday = new Date(`${today.getFullYear()}-09-29`) - today
```

Comment:
Until birthday is being calculated against 9-29 but the date check is for 8-29

Reply:
No it’s not, because months are zero-indexed in JS dates.